index.html

```diff
@@ -525,6 +521,10 @@ <h2>Privacy and Security</h2>
     be set to the same 5 microsecond minimum as the <code>Performance</code> interface
     defined in [[HR-TIME]].</p>
   </section>
+  <section>
+    <h2>Privacy Considerations</h2>
```

Comment:
I agree that the above risk holds both a security and a privacy risk. At the same time, it may be interesting to spell out the two different sides of that coin. (maybe as a followup)

Reply:
I guess this is more of a privacy consideration than a security one, as it's about time-based statistical fingerprinting.